src/services/AbstractDFG.jl

```diff
@@ -1059,7 +1059,7 @@ Returns a LightDFG{NoSolverParams, DFGVariableSummary, DFGFactorSummary}.
 """
 function getSummaryGraph(dfg::G)::LightDFG{NoSolverParams, DFGVariableSummary, DFGFactorSummary} where {G <: AbstractDFG}
     summaryDfg = LightDFG{NoSolverParams, DFGVariableSummary, DFGFactorSummary}(
-        description="Summary of $(dfg.description)",
+        description="Summary of $(getDescription(dfg))",
         userId=dfg.userId,
         robotId=dfg.robotId,
         sessionId=dfg.sessionId)
```

Comment:
Should user/robot/session should also use accessor?

Reply:
Knowing that every DFG struct has it and they will never be derived properties, I don't think so. Thoughts?